doc/source/reference/random/performance.rst

```diff
@@ -101,33 +101,33 @@ across tables.
 64-bit Linux
 ~~~~~~~~~~~~
 
-===================   =========  =======  ========  =======
-Distribution            MT19937    PCG64    Philox    SFC64
-===================   =========  =======  ========  =======
-32-bit Unsigned Int         100    119.8      67.7    120.2
-64-bit Unsigned Int         100    152.9      90.8    213.3
-Uniforms                    100    179.0      87.0    232.0
-Normals                     100    128.5      99.2    167.8
-Exponentials                100    148.3      93.0    189.3
-**Overall**                 100    144.3      86.8    180.0
-===================   =========  =======  ========  =======
+===================   =========  =======  ===========  ========  =======
+Distribution            MT19937    PCG64    PCG64DXSM    Philox    SFC64
+===================   =========  =======  ===========  ========  =======
+32-bit Unsigned Int         100      178          161        98      187
+64-bit Unsigned Int         100      177          183       111      213
+Uniforms                    100      197          203       115      230
+Normals                     100      125          121       102      160
+Exponentials                100      141          138        92      143
+**Overall**                 100      162          159       103      184
+===================   =========  =======  ===========  ========  =======
 
 
 64-bit Windows
 ~~~~~~~~~~~~~~
-The relative performance on 64-bit Linux and 64-bit Windows is broadly similar.
+The relative performance on 64-bit Linux and 64-bit Windows is broadly similar
+with the notable exception of the Philox generator.
 
-
-===================   =========  =======  ========  =======
-Distribution            MT19937    PCG64    Philox    SFC64
-===================   =========  =======  ========  =======
-32-bit Unsigned Int         100    129.1      35.0    135.0
-64-bit Unsigned Int         100    146.9      35.7    176.5
-Uniforms                    100    165.0      37.0    192.0
-Normals                     100    128.5      48.5    158.0
-Exponentials                100    151.6      39.0    172.8
-**Overall**                 100    143.6      38.7    165.7
-===================   =========  =======  ========  =======
+===================   =========  =======  ===========  ========  =======
+Distribution            MT19937    PCG64    PCG64DXSM    Philox    SFC64
+===================   =========  =======  ===========  ========  =======
+32-bit Unsigned Ints        100      147           90        27      147
+64-bit Unsigned Ints        100      144           72        24      173
+Uniforms                    100      149           75        24      171
+Normals                     100      133           94        37      158
+Exponentials                100      138           81        27      172
+**Overall**                 100      142           82        27      164
```

Comment:
Hmm, the Windows hand-"optimized" code path for DXSM needs some more work, then. I took the one with the 128*128 and just cut out the parts that weren't necessary for the 64-bit multiplier. Might need some more quality time with the [Compiler Explorer](https://godbolt.org/), then.